BUILDING.md

```diff
@@ -1,152 +1,147 @@
 ## BUILD INSTRUCTIONS AND REQUIREMENTS
 
-DB Browser for SQLite requires Qt as well as SQLite. For more information on Qt
-please consult http://www.qt.io and for SQLite please see https://sqlite.org/.
+DB Browser for SQLite requires Qt as well as SQLite.<br>
+For more information on Qt please consult http://www.qt.io and for SQLite please see https://sqlite.org/.
 
 Please note that all versions after 3.12.1 will require:
 * A C++ compiler with support for C++14 or later
-
-All versions after 3.9.1 will require:
-* Qt 5.5 or later, however we advise you to use 5.7 or later
-* A C++ compiler with support for C++11 or later
+* Qt 5.15.9 later
 
 Without these or with older versions you won't be able to compile DB Browser for
-Sqlite anymore. This applies to all platforms. However, most likely you won't
+SQLite anymore.<br>This applies to all platforms. However, most likely you won't
```

Comment:
Also, "anymore" should be "any more". :smile:

Reply:
Resolve via [043292c](https://github.com/sqlitebrowser/sqlitebrowser/pull/3485/commits/043292cb941a6d48870f60bc604c6e3445d7324b)